core/domain/subtopic_page_domain_test.py

```diff
@@ -230,54 +224,49 @@ def test_update_written_translations(self):
         }
 
         self.subtopic_page.update_page_contents_written_translations(
-            written_translations_dict)
+            written_translations_dict) # type: ignore[arg-type]
```

Comment:
Why is this ignore needed?

Reply:
This was done according to [this comment](https://github.com/oppia/oppia/pull/14955#discussion_r812007433).